components/single_image_view/single_image_view.tsx

```diff
@@ -217,6 +217,7 @@ export default class SingleImageView extends React.PureComponent<Props, State> {
                                 src={previewURL}
                                 dimensions={this.state.dimensions}
                                 fileInfo={this.props.fileInfo}
+                                fileURL={fileURL}
```

Comment:
Regarding my other comment, this might not be the URL we actually want to use here. Nothing to change here at the moment, but I wanted to make a note of it

Reply:
Coming back to this, the URL we're passing in is the private URL that requires you to have access to the channel to see, so that won't work here. I think we can just remove this prop then and make the `SizeAwareImage` responsible for loading the public link